overlay/index.js

```diff
@@ -73,8 +73,10 @@ function IframeRoot(document, root, props) {
   iframe.src = 'about:blank';
 
   iframe.style.border = 'none';
-  iframe.style.height = '100vh';
+  iframe.style.height = '100%';
   iframe.style.left = '0';
+  iframe.style.minHeight = '100vh';
+  iframe.style.minHeight = '-webkit-fill-available';
```

Comment:
Fixes an issue where the button panel would be hidden due to how `100vh` is calculated on iOS Safari.